pycbc/live/supervision.py

```diff
@@ -0,0 +1,134 @@
+# Copyright (C) 2023 Arthur Tolley, Gareth Cabourn Davies
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+#
+# =============================================================================
+#
+#                                   Preamble
+#
+# =============================================================================
+#
+""" This module contains functions for optimizing the signal-to-noise ratio
+of triggers produced by PyCBC Live. Also contained within this module are the
+command line arguments required and options group for the SNR optimization.
+This module is primarily used in the pycbc_optimize_snr program.
+"""
```

Comment:
Looks like the docstring needs to be rewritten.

Reply:
I also suggest removing the "preamble" box above. It makes the file longer and I never understood its purpose throughout other files in PyCBC and elsewhere.